test/plugin_functional/test_suites/saved_objects_management/visible_in_management.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { join } from 'path';
+import expect from '@kbn/expect';
+import type { Response } from 'supertest';
+import type { PluginFunctionalProviderContext } from '../../services';
+import { SavedObject } from '../../../../src/core/types';
+
+function parseNdJson(input: string): Array<SavedObject<any>> {
+  return input.split('\n').map((str) => JSON.parse(str));
+}
+
+export default function ({ getService }: PluginFunctionalProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  describe('types with `visibleInManagement` ', () => {
+    before(async () => {
+      await esArchiver.load(
```

Comment:
nit: It's not documented anywhere but the operations team wants everyone to move to using the import/export API instead of esArchiver 

Meta issue: https://github.com/elastic/kibana/issues/102552
PR introducing `kibanaServer.importExport` API for tests https://github.com/elastic/kibana/pull/92526